examples/index.js

```diff
@@ -74,6 +132,36 @@ async function getAccounts() {
   }
 }
 
+async function switchChain() {
+  console.log(chainIdToChange, 'switchChain')
+  try {
+    setRequest('wallet_switchEthereumChain')
+    const response = await provider.request({
+      method: 'wallet_switchEthereumChain',
+      params: [{ chainId: chainIdToChange }],
+    })
+    console.log({ response })
+  } catch (e) {
+    console.log(e)
+    console.log({ e })
```

Comment:
Need just one of these here. Same with the `addNetwork` function below.

Reply:
https://github.com/arcana-network/auth/pull/106/commits/efa2c361bc5be14ad15d797701f5aa27d18c1522